File: app-service-comments/src/main/java/com/github/arhor/aws/graphql/federation/comments/data/repository/PostRepresentationRepository.java
package com.github.arhor.aws.graphql.federation.comments.data.repository;

import com.github.arhor.aws.graphql.federation.comments.data.model.PostRepresentation;
import org.springframework.data.repository.CrudRepository;

import java.util.UUID;

public interface PostRepresentationRepository extends CrudRepository<PostRepresentation, UUID> {
}

File: app-service-votes/src/main/kotlin/com/github/arhor/aws/graphql/federation/votes/data/repository/PostRepresentationRepository.kt
package com.github.arhor.aws.graphql.federation.votes.data.repository

import com.github.arhor.aws.graphql.federation.votes.data.model.PostRepresentation
import org.springframework.data.repository.CrudRepository
import java.util.UUID

interface PostRepresentationRepository : CrudRepository<PostRepresentation, UUID>
